prime-router/src/main/kotlin/Report.kt

```diff
@@ -196,6 +197,17 @@ class Report : Logging {
                     else -> throw IllegalArgumentException("Unexpected extension $ext.")
                 }
             }
+
+            fun valueOfFromBodyFormat(bodyFormat: String): Format {
```

Comment:
Could this be replaced with `Format.valueOf()`?

Reply:
replaced with valueOfIgnoreCase()